librz/analysis/rzil/rzil.c

```diff
@@ -0,0 +1,98 @@
+#include <rz_analysis.h>
+
+/**
+ * Create an empty RzAnalysisRzil instance
+ * inner vm should be init in adaptive plugin
+ * @return RzAnalysisRzil* a pointer to RzAnalysisRzil instance
```

Comment:
Use `\return ` instead of `@return`, see https://github.com/rizinorg/rizin/blob/dev/DEVELOPERS.md#documentation